pkg/testutil/mock_test.go

```diff
@@ -0,0 +1,304 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package testutil
```

Comment:
I think testutil is no longer included for code coverage so probably no need to have unit test for mock.go.

Reply:
Yes, I understand. I feel that it makes sense to have these tests still.